lib/HALSim/LineSensors.cpp

```diff
@@ -95,34 +95,52 @@ void LineSensors::calibrate()
 
 int16_t LineSensors::readLine()
 {
-    const uint32_t WEIGHT0      = 0;
-    const uint32_t WEIGHT1      = 1000;
-    const uint32_t WEIGHT2      = 2000;
-    const uint32_t WEIGHT3      = 3000;
-    const uint32_t WEIGHT4      = 4000;
+    uint8_t        idx          = 0;
     uint32_t       estimatedPos = 0;
+    uint32_t       numerator    = 0;
+    uint32_t       denominator  = 0;
+    const uint32_t WEIGHT       = 1000;

```

Comment:
Until now, RadonUlzer didn't use the decorations, because the original code was for the sw-engineering part with the students. To keep the rule set small, at that time we decided to not force it.

But I will create an issue to update this for ALL modules.

Reply:
See #79